src/gui/SettingsWidget.cpp

```diff
@@ -123,6 +125,11 @@ void SettingsWidget::loadSettings()
     m_generalUi->autoTypeEntryTitleMatchCheckBox->setChecked(config()->get("AutoTypeEntryTitleMatch").toBool());
     m_generalUi->autoTypeEntryURLMatchCheckBox->setChecked(config()->get("AutoTypeEntryURLMatch").toBool());
     m_generalUi->ignoreGroupExpansionCheckBox->setChecked(config()->get("IgnoreGroupExpansion").toBool());
+    m_generalUi->autoHideInternalMessagesCheckBox->setChecked(config()->get("AutoHideInternalMessages", false).toBool());
+    m_generalUi->autoHideInternalMessagesTimeoutSpinBox->setValue(
```

Comment:
This should be in seconds not milliseconds.

Reply:
ok :ok_hand: